### ISSUE TITLE
Distinction between each of the components required for Solid

### ISSUE THREAD

**Issue by tomhgmns** [solid/test-suite#143]
Hi all,

The goal of the Solid standard is to make sure that there is a loose coupling between each of the components required for Solid (e.g. storage, UMA AS, IDP, Authz Agent, ...). For example, on use.id, everyone can choose which store, IDPs, ... to connect with their WebID.

However, currently the test-suite reports raise the impression that every component (storage, UMA AS, IDP, Authz Agent, ...) required for Solid should be encapsulated in a single software suite (see the image below) while this is absolutely not the case.

Therefore, I was wondering whether this could be made more clear in the reports.

One option would be to distinguish between each of the components and create tests for each of those. For example, one could create a list of Solid compliant IDPs and evaluate whether they comply with Solid-OIDC.

<img width="927" alt="image" src="https://user-images.githubusercontent.com/17943609/170213247-cf8a2f9a-b1aa-4214-914c-4b39cf311f17.png">

**Comment by csarven** [comment]
Thanks. Acknowledging the point that summaries of reports, as well as reports for a set of tests, can be done in different ways and levels. I believe the the interest in reporting on individual tests is captured in this issue https://github.com/solid/test-suite-panel/issues/5 and will be communicated in the upcoming charter and the work items.

If you're satisfied with this answer, can we close this issue?

**Comment by tomhgmns** [comment]
Thanks! I will mention this issue there as well and you can close this one :-).

**Comment by michielbdejong** [comment]
@tomhgmns good point! Especially to the point where someone could compose a spec-compliant Solid server by "shopping around" for the various components from different projects. We will take it into account! Thanks for pointing it out.

**Comment by csarven** [comment]
Oh! Pardon me. I was under the assumption that this issue is raised in the solid/test-suite-panel so I was mistaken to suggest to close this issue. I thought that this is a duplicate issue or already covered by the issue in the test-suite-panel (re work on charter). 

Reopening so that it can be followed up.
